travis.sh

```diff
@@ -127,7 +127,7 @@ if [ "$USE_DOCKER" = true ]; then
     -e ROS_PARALLEL_JOBS -e ROS_PARALLEL_TEST_JOBS \
     -e ROSDEP_ADDITIONAL_OPTIONS -e ROSDEP_UPDATE_QUIET \
     -e SUDO_PIP -e USE_PYTHON_VIRTUALENV \
-    -e NOT_TEST_INSTALL \
+    -e NOT_TEST_INSTALL -e DISTRO \
```

Comment:
Sorry, but could you use `` `lsb_release -cs` `` to set `DISTRO` inside docker, instead of passing variable?
https://github.com/jsk-ros-pkg/jsk_travis/blob/0.5.6/travis.sh#L181
I want to keep passing variables minimum.

Reply:
I think this is already “fixed” https://github.com/docker-library/official-images/pull/7029#issuecomment-559202024

Reply:
if the docker image is fixed, it does not matter.

Reply:
i checked it and docker image is not fixed.